src/cljs/nr/utils.cljs

```diff
@@ -292,12 +292,7 @@
 (defn render-cards
   "Render all cards in a given text or HTML fragment input"
   [input]
-  (cond
```

Comment:
this function was only ever called by `render-message` - I'm not really sure there's a need to check contains-card-pattern anymore as the card-patterns are now memoized.  Additionally `re-find` will error against non-strings so some of the branches in this condition didn't even make sense.